ldk/go/whisperContentDisambiguation.go

```diff
@@ -4,6 +4,7 @@ import "github.com/open-olive/loop-development-kit/ldk/go/proto"
 
 type WhisperContentDisambiguation struct {
 	Label    string                                         `json:"label"`
+	Markdown string											`json:"markdown"`
```

Comment:
Pretty sure golint is gonna freak out about this.

Reply:
Yeah, you're right. I go fmted the code